### PR TITLE
Tts en tech terms

### PR DESCRIPTION
**Pull request by mgrafu**
# What does this PR do ?

changes to en tn for tts

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Have you signed your commits? Use ``git commit -s`` to sign.
- [x] Do all unittests finish successfully before sending PR?
   1) ``pytest`` or (if your machine does not have GPU) ``pytest --cpu`` from the root folder (given you marked your test cases accordingly `@pytest.mark.run_only_on('CPU')`).
   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
- [ ] If you are adding a new feature: Have you added test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
- [ ] Have you added ``__init__.py`` for every folder and subfolder, including `data` folder which has .TSV files?
- [ ] Have you followed codeQL results and removed unused variables and imports (report is at the bottom of the PR in github review box) ?
- [ ] Have you added the correct license header `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` to all newly added Python files?
- [ ] If you copied [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
- [ ] Remove import guards (`try import: ... except: ...`) if not already done.
- [ ] If you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
- [ ] Have you added your language support to [tools/text_processing_deployment/pynini_export.py](tools/text_processing_deployment/pynini_export.py).
  

**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation
- [ ] Test

If you haven't finished some of the above items you can still open "Draft" PR.

### CODE REVIEW

**Review by tbartley94 — DISMISSED** [review]

File: nemo_text_processing/text_normalization/en/taggers/electronic.py [hunk 1]
```diff
@@ -118,6 +120,13 @@ def __init__(self, cardinal: GraphFst, deterministic: bool = True):
         # www.abc.com/sdafsdf, or https://www.abc.com/asdfad or www.abc.abc/asdfad
         graph |= protocol + pynutil.insert(" ") + domain_graph_with_class_tags
 
+        # credit card cues
+        numbers = pynini.closure(NEMO_DIGIT, 4, 16)
```
[the remark this answers is not in the document]
Comment (mgrafu) [reply nemo_text_processing/text_normalization/en/taggers/electronic.py:124]: discussed offline

**Comment by tbartley94** [comment]
Commenting for clarity on review: cardinals and credit cards are going to overlap here. We can permit a higher weight for full credit card strings `xxx xxx xxx xxx` but the abbreviated four digit string `xxxx` should only be caught by the non-deterministic graph.

**Comment by tbartley94** [comment]
@mgrafu 

`ASSERT:1:01:01 expected:<one hour one minute and one second> but was:<one hour one minutes and one seconds>`

Plurals for time was altered.

**Review by tbartley94 — APPROVED** [review]
LGTM

**Pull request merged**